src/main/java/org/apache/maven/plugins/checkstyle/exec/DefaultCheckstyleExecutor.java

```diff
@@ -201,8 +201,6 @@ public URLClassLoader run()
             }
         } );
 
-        checker.setClassLoader( projectClassLoader );
```

Comment:
Variable `projectClassLoader` still remains and all it does it call close a few lines down. Seems to me it can be completely removed now.

Reply:
Sure, sorry for the oversight.